### DISCUSSION TITLE
v7.2.13 Release of PowerShell

### DISCUSSION THREAD

**Discussion by adityapatwardhan** [PowerShell/PowerShell#19965] [General]
### Tests

- Increase the timeout to make subsystem tests more reliable (#19937)
- Increase the timeout when waiting for the event log (#19936)

### Build and Packaging Improvements

<details>

<summary>

<p>Bump .NET SDK version to 6.0.412</p>

</summary>

<ul>
<li>Update Notice file (#19956)</li>
<li>Update cgmanifest (#19938)</li>
<li>Bump to 6.0.412 SDK (#19933)</li>
<li>Update variable used to bypass the blocking check for multiple NuGet feeds (#19935)</li>
</ul>

</details>

[7.2.13]: https://github.com/PowerShell/PowerShell/compare/v7.2.12...v7.2.13

### SHA256 Hashes of the release artifacts

- hashes.sha256
  - 57FB8BF41F4066946152A7DDDE6F5C9EB2C663D9930A5DF030A0A98DE69F11CF
- powershell_7.2.13-1.deb_amd64.deb
  - 0AF024461C5184A387FB346EE00B3DA3DA5FA734CE7BDEEC65B085870CA4EA7E
- powershell-7.2.13-1.cm.x86_64.rpm
  - AC4A7D2032488526401AA8D4A93AE9262C6D258AA3D6A12B31BAA33B28EF7EDD
- powershell-7.2.13-1.rh.x86_64.rpm
  - 4F2946F07355276F09F08A7D8F39F53528441893D5082EABCF6C5AB97D208A4B
- powershell-7.2.13-linux-alpine-x64.tar.gz
  - 8518ABD0EF7BBC12EA10B78DB6B730EAB89CABE2166279BBEF7BC9C4BB6FE3AC
- powershell-7.2.13-linux-arm32.tar.gz
  - F891B4663FC452562DD6D4713DEAA1B72D3A8154F58455049134863348DFCBA1
- powershell-7.2.13-linux-arm64.tar.gz
  - FB1CD6D11C4AB01FD8380239FE85D332DE542E8DD1B5592512D3D6C1F5B4F9FB
- powershell-7.2.13-linux-x64.tar.gz
  - BB1FD3924D5A81A7E9C775A1794FA1766AE57F87F1560B24344FAD58600C5ED0
- powershell-7.2.13-linux-x64-fxdependent.tar.gz
  - 72921B12FEC83946555757F394EFFAA8CA6482E0DBE278AE6502319A8D3B31F8
- powershell-7.2.13-osx-arm64.pkg
  - 43C3C6EF49FA5A5ED8BCC48E26335FCB0BC66E4CA2F682E9A545EB06E63CC716
- powershell-7.2.13-osx-arm64.tar.gz
  - FCECB7CD7C5444BC2DAA771236B61C96E797A470FE5206950F27136B342531FD
- powershell-7.2.13-osx-x64.pkg
  - 5909A7D3DBC8200BF170243C02D7DC2C6239899D6D5E76B18FCB84832DB7F857
- powershell-7.2.13-osx-x64.tar.gz
  - C733A14AE5683E166A4D8925370BB02B3DCF8E835CB7DA429BF9651DA3DA2519
- PowerShell-7.2.13-win-arm32.zip
  - 181D2B727BB27C796E4FD4B4AE755FD7888D30408F2C94A401AA2A6557C13E0A
- PowerShell-7.2.13-win-arm64.zip
  - 993AD7D38B4C139B3E2BA9C15B4D78DE0E478D64FF2392F0C520AF49FE932839
- PowerShell-7.2.13-win-fxdependent.zip
  - 1A8B6DE5377DA1533F607BB10FBDE4F248AC3A7E16137C3C862A2245AE351B2F
- PowerShell-7.2.13-win-fxdependentWinDesktop.zip
  - 1785D94D4A04C660AE3236BA07D1FF065D39106E5B45989A2E2ABDCB685B35D0
- PowerShell-7.2.13-win-x64.msi
  - E1A2059D91099ABDE5E469A44EEBF3242891CBED93043D05491174F8510B2D03
- PowerShell-7.2.13-win-x64.zip
  - E81D266716598DA09727A604887AC57EF3A2553CBE62344CA5AC9309B9386AD1
- PowerShell-7.2.13-win-x86.msi
  - 7F5A01A5D24A597674D35DE20759764441226D052CDE6B529C018C81FB172771
- PowerShell-7.2.13-win-x86.zip
  - 66D9A5C0C48C0DA826DA3B2DFA309EDCD2B43547096838634258941C6E6B66D4
- powershell-lts_7.2.13-1.deb_amd64.deb
  - 75A1204F0853C30057405DCA9F725FB28D2A6114C217B501C3A541D1EAF5ED92
- powershell-lts-7.2.13-1.cm.x86_64.rpm
  - F179A4AAC3B82169A4E5777508DFE9390B29D09ADB844A3452B88DFD9B290ABB
- powershell-lts-7.2.13-1.rh.x86_64.rpm
  - BB9229AE61614B26CE54E0B142DE43EF4115C40C5969B5A7D6EFA62AF88B6887
- powershell-lts-7.2.13-osx-arm64.pkg
  - CDB881D5A624E6986C0D5340C1850AC155597B8A918D6674885C66C7EB7729D6
- powershell-lts-7.2.13-osx-x64.pkg
  - 359E50F53C761B9EB5B8B70E3439C4CA2460A57CCDC96693669541052F78FD55

<hr /><em>This discussion was created from the release <a href='https://github.com/PowerShell/PowerShell/releases/tag/v7.2.13'>v7.2.13 Release of PowerShell</a>.</em>

**Comment by Ric1623** [comment]
Go
